osl/maxfilter/maxfilter.py

```diff
@@ -480,7 +483,7 @@ def run_cbu_3stage_maxfilter(infif, outbase, args):
 
 
 def main(argv=None):
-
+    print('HELLO!?')
```

Comment:
I think this was a debugging message you forgot to remove?

Reply:
Fixed in https://github.com/OHBA-analysis/oslpy/pull/48/commits/600ab9032269cc4e12ed28c8d5171ba0112de2cb